R/datamapping-xy.R

```diff
@@ -11,17 +11,13 @@ XYDataMapping <- R6::R6Class(
     y = NULL,
     data = NULL,
 
-    initialize = function(x, y) {
+    initialize = function(x, y = NULL) {

```

Comment:
How can Y be null for a datamapping xy? ?

Reply:
The histogram takes only an x.  I had the choice of either making a datamapping-x class plus a datamapping-x-group class or reuse datamapping-xy-grouping but give y a null default.   I think the latter might be less cluttered?

Reply:
Maybe a not relevant remark, for the datamapping-x-group class:
Why not inheriting directly from datamapping-xy-grouping, and then the initialize function won't take y as input ? So, you can call super$initialize with directly y=NULL.
(or x, I was thinking that the plotted variable would be called why in the histogram)

Reply:
I think we should reuse. As long as derived class enfore a y to be present, we are good